Feature capstone project 1

kit0-0 opened the pull request:
**This pull request includes the following changes to the Capstone project:**

- [ ]  Add Home page (Mobile/Desktop).
- [ ]  Add About page (Mobile/Desktop).
- [ ] Added CSS for Home and About pages
- [ ] Implemented dynamic elements on the Home page ( speaker and Program section).

These changes greatly improve the overall user experience and functionality of the Capstone project.

NahnahAJ reviewed (changes requested):
## Project Status: CHANGES REQUIRED :recycle: 
Hi @kit0-0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/m3864rBwwBTKMAbpn3/giphy.gif"/>

## Highlights
- Very nice design :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
- Professional README :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kit0-0 commented:
# New additions:

- Changed the background of the mobile menubar.
- Updated the heading on the home page specifically for the mobile version.
- Added italic style to the "Featured Speaker" element.
- Applied italic style and increased the font size to the "speaker-header," "main section heading," and "main-sec-heading-about" elements. Additionally, made the main section heading bold for better emphasis.
- Unfortunately, a suitable background picture for the main section heading could not be found.
- Resized the past projects section.

Zuhaib042 commented on style.css:
```diff
@@ -0,0 +1,846 @@
+}
+
+@media (min-width: 768px) {
+  .about-container {
+    background: url(./images/about_bg_01.png);
+  }
```
- [ ] As suggested by the previous reviewer, consider adding `background-color: #f7f7f7`to .about-container on all screen sizes to have light gray background-color in the about section.

Zuhaib042 commented on about.html:
```diff
@@ -0,0 +1,102 @@
+        </div>
+    </main>
+    <section class="section-2">
+        <h2 class="about-h2">Creative city Global Summit 2015 Logo</h2>
```
- [OPTIONAL] Consider giving this element more `margin-top` to make it similar to the original design.

Zuhaib042 commented on about.html:
```diff
@@ -0,0 +1,102 @@
+    </section>
+
+    <section class="section-3">
+        <h2 class="about-h2">View Past Global Summits</h2>
```
- [OPTIONAL] Consider giving this element more `margin-top` to make it similar to the original design.

Zuhaib042 commented on index.html:
```diff
@@ -0,0 +1,97 @@
+
+    </section>
+    <!-- sponsers -->
+    <section class="partner hidden">
```
- [OPTIONAL] Kindly consider adding this background-color `#414246` to this section, and then `gray` color to the logos to make it similar to the behance design.

Zuhaib042 commented on about.html:
```diff
@@ -0,0 +1,102 @@
+    </section>
+
+    <!-- sponsers -->
+    <section class="partner about-partner">
```
- [OPTIONAL] Kindly consider adding this background-color #414246 to this section, and then gray color to the logos to make it similar to the behance design.

kit0-0 commented:
# New additions:
- Added background-color: #f7f7f7 to the .about-container class.
- Applied margin-top to section 2 and section 3 for proper spacing.
- Set the background-color to #414246 for the partner section.
- Modified the logo colors to have a consistent color throughout.

ichala reviewed (approved):
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966](https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif)

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
